vss-core/src/test/java/org/eclipse/kuksa/vsscore/extension/StringExtensionTest.kt

```diff
@@ -14,14 +14,22 @@
  * limitations under the License.
  *
  * SPDX-License-Identifier: Apache-2.0
- *
  */
 
-package org.eclipse.kuksa.testapp.extension
+package org.eclipse.kuksa.vsscore.extension
 
-import org.eclipse.kuksa.extension.dataPointValueCase
-import org.eclipse.kuksa.proto.v1.Types
-import org.eclipse.kuksa.proto.v1.Types.Datapoint.ValueCase
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
 
-val Types.Metadata.valueType: ValueCase
-    get() = dataType.dataPointValueCase
+class StringExtensionTest : BehaviorSpec({
+    given("Multiple strings") {
+        val testStrings = listOf("VSSVehicleID", "vehicleIDENTIFIER", "VVehicleABvolume")
+        `when`("converting it to camelcase") {
+            val camelCaseStrings = testStrings.map { it.toCamelCase }
+            then("it should be in a valid camelcase format") {
+                val expectedCamelCases = listOf("vssVehicleID", "vehicleIdentifier", "vVehicleABvolume")
+                camelCaseStrings shouldBe expectedCamelCases
+            }
```

Comment:
I would add a separate "test case" for the special case of "not lower-casing 2-letter abbreviations" and / or add a link to the source. Otherwise it might feel cheated / arbitrary why we decided for this.